CHANGELOG.md

```diff
@@ -100,6 +100,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Unicode 13 support
 - Option to run command on bell which can be set in `bell.command`
 - Fallback to program specified in `$SHELL` variable on Linux/BSD if it is present
+- Ability to make selections while search is active
```

Comment:
I've also noticed that you've fixed one minor issue just by enabling our common mouse interactions while search is active.

We were highlighting url in search mode, but we were blocking the opening, however now it opens as expected, so I guess we should mention that, since if blocking url opening was expected we shouldn't highlight it in a first place.